packages/next/client/image.tsx

```diff
@@ -222,6 +252,8 @@ export default function Image({
   objectFit,
   objectPosition,
   loader = defaultImageLoader,
+  placeholder = defaultPlaceholder,
```

Comment:
I think the default should be `empty`.

Then in the function body, you can override the user's value if `configEnableBlurryPlaceholder` is not enabled.

```js
if (!configEnableBlurryPlaceholder) {
  placeholder = 'empty'
}
```

That way, you only check `configEnableBlurryPlaceholder` once instead of multiple times.

Reply:
Made the change